doc/requirements.txt

```diff
@@ -2,7 +2,8 @@ Image==1.5.30
 PyYAML>=4.2b1
 cmake==3.26.0
 csvkit==1.1.1
-cython==0.29.0
+# require cython version handling removal of tp_print with Python 3.9 correctly
```

Comment:
```suggestion
```
I'd rather not have this comment here, as it will soon sound totally meaningless and we're probably not ever going back to older version here. No one ever cleans these things up in the future...

Reply:
I guess no-one will set this back to an older Cython version, so I removed the comment.